src/libs/Navigation/DeprecatedCustomActions.js

```diff
@@ -102,16 +102,23 @@ function pushDrawerRoute(route) {
         const screenRoute = {type: 'route', name: newScreenName};
         const history = _.map(state.history ? [...state.history] : [screenRoute], () => screenRoute);
 
-        // Force drawer to close
-        // https://github.com/react-navigation/react-navigation/blob/94ab791cae5061455f036cd3f6bc7fa63167e7c7/packages/routers/src/DrawerRouter.tsx#L142
-        const hasDrawerhistory = _.find(state.history || [], h => h.type === 'drawer');
-        if (!hasDrawerhistory || currentState.type !== 'drawer') {
+        const drawerHistoryItem = _.find(state.history || [], h => h.type === 'drawer');
```

Comment:
More context about the changes in `DeprecatedCustomActions.js` [this thread](https://expensify.slack.com/archives/C01GTK53T8Q/p1664404003728569)